docs/src/main/sphinx/optimizer/cost-based-optimizations.md

```diff
@@ -15,27 +12,26 @@ the query's execution, then subsequent stages need to process large amounts of
 data for longer than necessary, increasing the time and resources needed for
 processing the query.
 
-With cost-based join enumeration, Trino uses :doc:`/optimizer/statistics`
+With cost-based join enumeration, Trino uses {doc}`/optimizer/statistics`
 provided by connectors to estimate the costs for different join orders and
 automatically picks the join order with the lowest computed costs.
 
-The join enumeration strategy is governed by the ``join_reordering_strategy``
-:ref:`session property <session-properties-definition>`, with the
-``optimizer.join-reordering-strategy`` configuration property providing the
+The join enumeration strategy is governed by the `join_reordering_strategy`
+{ref}`session property <session-properties-definition>`, with the
+`optimizer.join-reordering-strategy` configuration property providing the
 default value.
 
 The possible values are:
 
- * ``AUTOMATIC`` (default) - enable full automatic join enumeration
- * ``ELIMINATE_CROSS_JOINS`` - eliminate unnecessary cross joins
- * ``NONE`` - purely syntactic join order
+> - `AUTOMATIC` (default) - enable full automatic join enumeration
+> - `ELIMINATE_CROSS_JOINS` - eliminate unnecessary cross joins
+> - `NONE` - purely syntactic join order
```

Comment:
I guess the original block is rendered as a quote unintentionally. We can change to normal bulleted list in a follow-up. Same for other places.

Reply:
Yeah .. after the conversion I full intend to use some more markdown native syntax where I can and improve these kind of smaller issues.